src/idm/group.js

```diff
@@ -0,0 +1,125 @@
+import axios from 'axios';
+import { errorHandler } from '../utils';
+
+const group = (base, token) => {
+  base = `${base}`;
+  return ({
+    /**
+    * @summary List all groups
+    * @name getGroups
+    * @public
+    * @function
+    * @memberof agile.idm.group
+    * @fulfil {Array} all groups
+    * @returns {Promise}
+    * @example
+    * agile.idm.group.getGroups().then(function(groups) {
```

Comment:
I am confused because I see that you wrote the same call twice, i.e. agile.idm.grou.get(). 
But, after checking de device api, I have added the arguments to get a particular group with get. However, I have left the possibility to call it without arguments to get all the groups. I am not sure whether this is an issue for the API still... 
What you meant before is that we should have a  groupManager namespace with just one call to get the groups? or would it fit as it is now?